pyanaconda/core/constants.py

```diff
@@ -390,3 +390,9 @@ class PayloadRequirementType(Enum):
 PARTITIONING_METHOD_MANUAL = "MANUAL"
 PARTITIONING_METHOD_INTERACTIVE = "INTERACTIVE"
 PARTITIONING_METHOD_BLIVET = "BLIVET"
+
+# Payload URL source types.
+URL_SOURCE_BASE = "BASE"
```

Comment:
I would call it `URL_SOURCE_BASEURL` and `BASEURL` to keep the names same as in DNF.

Reply:
Fixed.